google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddResponsiveSearchAdWithAdCustomizer.java

```diff
@@ -0,0 +1,269 @@
+import com.google.ads.googleads.v9.enums.CustomizerAttributeTypeEnum.CustomizerAttributeType;
+import com.google.ads.googleads.v9.errors.GoogleAdsError;
+import com.google.ads.googleads.v9.errors.GoogleAdsException;
+import com.google.ads.googleads.v9.resources.Ad;
+import com.google.ads.googleads.v9.resources.AdGroupAd;
+import com.google.ads.googleads.v9.resources.CustomerCustomizer;
+import com.google.ads.googleads.v9.resources.CustomizerAttribute;
+import com.google.ads.googleads.v9.services.CustomerCustomizerOperation;
+import com.google.ads.googleads.v9.services.CustomerCustomizerServiceClient;
+import com.google.ads.googleads.v9.services.CustomizerAttributeOperation;
+import com.google.ads.googleads.v9.services.CustomizerAttributeServiceClient;
+import com.google.ads.googleads.v9.services.MutateCustomerCustomizersResponse;
+import com.google.ads.googleads.v9.services.MutateCustomizerAttributesResponse;
+import com.google.common.collect.ImmutableList;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Adds a customizer attribute, links the customizer attribute to a customer, and then adds a
+ * responsive search ad with a description using the ad customizer to the specified ad group.
+ */
+public class AddResponsiveSearchAdWithAdCustomizer {
+
+  private static class AddResponsiveSearchAdWithAdCustomizerParams extends CodeSampleParams {
+
+    @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+    private Long customerId;
+
+    @Parameter(names = ArgumentNames.AD_GROUP_ID, required = true)
+    private Long adGroupId;
+
+    // The name of the customizer attribute to be used in the ad customizer, which must be unique.
+    // To run this example multiple times, change this value or specify its corresponding argument.
+    // Note that there is a limit for the number of enabled customizer attributes in one account,
+    // so you shouldn't run this example more than necessary.
+    // Visit
+    // https://developers.google.com/google-ads/api/docs/ads/customize-responsive-search-ads?hl=en#rules_and_limitations
+    // for details.
+    @Parameter(names = ArgumentNames.CUSTOMIZER_ATTRIBUTE_NAME)
+    private String customizerAttributeName;
+  }
+
+  public static void main(String[] args) {
+    AddResponsiveSearchAdWithAdCustomizerParams params =
+        new AddResponsiveSearchAdWithAdCustomizerParams();
+    if (!params.parseArguments(args)) {
+
+      // Either pass the required parameters for this example on the command line, or insert them
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+      params.adGroupId = Long.parseLong("INSERT_AD_GROUP_ID_HERE");
+
+      // Optional: Specify a customizerAttributeName. The default value in this example is "Price".
+      params.customizerAttributeName = "Price";
```

Comment:
This line doesn't work if run the example when only providing **--customerId** and **--adGroupId** params, e.g.

./gradlew -q runExample --example="advancedoperations.AddResponsiveSearchAdWithAdCustomizer --customerId 1111111111 --adGroupId 2222222222"

Reply:
Here is an example for similar case

https://github.com/googleads/google-ads-java/blob/main/google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddAdGroupBidModifier.java#L51